internal/logging/factory.go

```diff
@@ -8,7 +8,7 @@ import (
 func Configure() {
 	handlerOptions := &slog.HandlerOptions{
 		AddSource: true,
-		Level:     slog.LevelError,
+		Level:     slog.LevelInfo,
```

Comment:
I think this should default to Error but with an env var to change the level
The CLI prints the output from the tools (packer, terraform, and ansible), and this is of interest to the user. Additional logging of what the CLI is doing is not of interest on the happy path